lte/alt1250/alt1250_netdev.c

```diff
@@ -60,7 +60,7 @@ void alt1250_netdev_unregister(FAR struct alt1250_s *dev)
 
 void alt1250_netdev_ifdown(FAR struct alt1250_s *dev)
 {
-  dev->net_dev.d_flags = IFF_DOWN;
+  dev->net_dev.d_flags = ~IFF_UP;
```

Comment:
@xiaoxiang781216 
Sorry for the delay, let me confirm.
Is this as you intended? Or is it a mistake?
`dev->net_dev.d_flags &= ~IFF_UP;`

Reply:
Yes, it's intended to follow how other POSIX OS turn off netdev. After IFF_DOWN is removed, ~IFF_UP represent down.